src/lib/fillers/__tests__/fetchCollectorSignals.test.ts

```diff
@@ -0,0 +1,136 @@
+import { fetchCollectorSignals } from "lib/fillers/fetchCollectorSignals"
+import { isFeatureFlagEnabled } from "lib/featureFlags"
+
+jest.mock("lib/featureFlags", () => {
+  return {
+    isFeatureFlagEnabled: jest.fn(),
+  }
+})
+
+const mockIsFeatureFlagEnabled = isFeatureFlagEnabled as jest.Mock
+
+// Mock context and artwork setup
+const ctx = {
+  userID: "testUser",
+  mePartnerOffersLoader: jest.fn(),
+  salesLoader: jest.fn(),
+  saleArtworkLoader: jest.fn(),
+}
+
+const artwork = {
+  _id: "artwork1",
+  sale_ids: ["sale1"],
+  purchasable: true,
+  recent_saves_count: 10,
+}
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
```

Comment:
Can we move these above definitions into the `describe` block and out of the global scope?